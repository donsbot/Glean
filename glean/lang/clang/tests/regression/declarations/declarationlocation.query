query: |
    { N, L } where cxx1.DeclarationLocationName { _, N, L }
transform: [gensort, []]
