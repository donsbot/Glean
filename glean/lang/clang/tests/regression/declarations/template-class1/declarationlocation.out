[
  "@generated",
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 10,
        "columnBegin": 1,
        "lineEnd": 10,
        "columnEnd": 8
      },
      "tuplefield1": "S"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 13,
        "columnBegin": 1,
        "lineEnd": 20,
        "columnEnd": 1
      },
      "tuplefield1": "S"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 14,
        "columnBegin": 3,
        "lineEnd": 14,
        "columnEnd": 16
      },
      "tuplefield1": "constructor"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 14,
        "columnBegin": 5,
        "lineEnd": 14,
        "columnEnd": 12
      },
      "tuplefield1": ""
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 15,
        "columnBegin": 3,
        "lineEnd": 15,
        "columnEnd": 11
      },
      "tuplefield1": "constructor"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 15,
        "columnBegin": 5,
        "lineEnd": 15,
        "columnEnd": 7
      },
      "tuplefield1": ""
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 17,
        "columnBegin": 3,
        "lineEnd": 19,
        "columnEnd": 3
      },
      "tuplefield1": "U"
    }
  }
]